### PR TITLE
Added process of quotes

### PR DESCRIPTION
**Pull request by jaumarar**
If the .env file has a value wrapped in single or double quotes it will be parsed into a simple string:

.env file:
VARIABLE="value with double quotes"

Before:
$_ENV['VARIABLE] = "value with double quotes";

Now:
$_ENV['VARIABLE] = value with double quotes;

Also:
- Separate the process of variables and the configuration to other classes
- Added test to this feature
- Modified README

### CODE REVIEW

File: src/Option.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace DevCoder;
+
+class Option
```
Comment (michelphp) [inline src/Option.php]: can you change the class by an interface ? and this interface will be implemented by processor class

File: src/Processor.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace DevCoder;
+
+class Processor
```
Comment (michelphp) [inline src/Processor.php]: Can you create a class unique for one resolve ? 
in the Processor class, create a collection of Resolver and each resolver resolves a value
thx

**Comment by jaumarar** [comment]
Hi! @devcoder-xyz,
had you have time to look at this propossal? My intention is to use this package in several projects and it would be nice to add features like this and the process of numbers, for example. 

I'm glad to help and hear your opinion about this so if there is something that you are not comfortable adding there will be no problems. Greetings

**Comment by michelphp** [comment]
@jaumarar Thanks for your help, had you read my comments ? i submit review if you can fix it

**Comment by jaumarar** [comment]
Hi @devcoder-xyz i'm not seeing any reviews or comments in the mr nor other branches. 

can you link me here those comments? maybe are marked as draft or the reviews has not been started

**Comment by michelphp** [comment]
ok no problem :+1: 
Processor.php : Can you create a class unique for one resolve ?
in the Processor class, create a collection of Resolver and each resolver resolves a value

its possible ?

**Comment by jaumarar** [comment]
Sure, something like?
```
Processor.php
Processor
  | BooleanProcessor.php
  | QuotedProcessor.php
```
Where Processor.php

```php
class Processor
{
  execute($value)
  {
    foreach ([BooleanProcessor::class, QuotedProcessor::class] as $processorClass) {
      $processor = new $processorClass($value);

      if ($processor->canBeExecuted()) {
        return $processor->execute();
      }
    }

    return $value;
  }
}
```

**Comment by michelphp** [comment]
yes like this but Processor's Name willl be added in DotEnv  (constructor)

**Comment by jaumarar** [comment]
I think it should be finished now

**Review by michelphp — COMMENTED** [review]
# src/DotEnv.php
you forget "?" after array type,
Default value is NULL its ok but if $processors is null set Processors(null) , There will be a fatal error

# src/Processor/QuotedProcessor.php

isWrappedByChar can be a private method

thank you very much @jaumarar  it's perfect

**Comment by jaumarar** [comment]
Hi @devcoder-xyz should I change anything more?

**Comment by michelphp** [comment]
Yes please , you not see my comment ?

src/DotEnv.php

you forget "?" after array type,
Default value is NULL its ok but if $processors is null set Processors(null) , There will be a fatal error
src/Processor/QuotedProcessor.php

isWrappedByChar can be a private method

thank you very much @jaumarar it's perfect

**Comment by jaumarar** [comment]
Hi @devcoder-xyz I was away for a long time..

I hace added a test to proof that that `DotEnv::setProcessors()` should be safe to pass anything invalid since it filters invalid values, let me know if you need anything more

**Pull request merged**